lib/travis/api/v3/services/allowance/for_owner.rb

```diff
@@ -0,0 +1,15 @@
+module Travis::API::V3
+  class Services::Allowance::ForOwner < Service
+    def run!
+      return result BillingClient.default_allowance_response if Travis.config.org?
+      raise LoginRequired unless access_control.logged_in?
+
+      owner = query(:owner).find
+
+      raise NotFound unless owner
+      # raise InsufficientAccess unless access_control.visible?(owner)
```

Comment:
seems like it can be removed

Reply:
Of course, thanks @pavel-d for noticing.